src/main/routes/scenarios.js

```diff
@@ -1,6 +1,16 @@
 const express = require('express');
 const router = express.Router();
 
+//Route firebaseAdmin
+const admin = require('firebase-admin');
+
+admin.initializeApp({
```

Comment:
The same: Please create an isolation file with something like this:

```
const admin= const admin = require('firebase-admin');
export const db = admin.initializeApp({
    credential: admin.credential.applicationDefault(),
    databaseURL: 'https://metal-slug-maker-default-rtdb.firebaseio.com'
});
```

Reply:
Yeah, i will check it

Reply:
The thing is that the db will be used in your get or post router, so i think it´s not that necessary for this iteration